code/game/machinery/doors/poddoor.dm

```diff
@@ -59,9 +59,18 @@
 /obj/machinery/door/poddoor/try_to_activate_door(mob/user)
  	return
 
-/obj/machinery/door/poddoor/try_to_crowbar(obj/item/I, mob/user)
+/obj/machinery/door/poddoor/try_to_crowbar(mob/user, obj/item/I)
+	if(!density)
+		return
 	if(!hasPower())
-		open()
+		to_chat(user, "<span class='notice'>You start forcing [src] open...</span>")
+		if(do_after(user, 50 * I.toolspeed, target = src))
```

Comment:
can put hasPower in as an extra check for the do_after. This will stop the do_after from working if the door regains power. Can also remove the line below then

Reply:
If I do this it will remove user feedback on the fact the pod door is powered again, which is why the forcing open failed. Or I will need to check if door is powered to show a message or not which defeats the point of the change... Keeping it as is.

Reply:
You can still have the feedback. Just move it down into the else of the `do_after`. That way the user won't do work while it won't result in anything and the user is notified on why it failed